host/4/publish-appservice-stage.yml

```diff
@@ -225,20 +225,25 @@ steps:
       docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.9-appservice
       docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.10-appservice
       docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.11-appservice
+      docker pull $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.12-appservice
+
 
       docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.7-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.7-appservice-stage$(StageNumber)
       docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.8-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.8-appservice-stage$(StageNumber)
       docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.9-appservice $TARGET_REGISTRY/python:$(TargetVersion)-appservice-stage$(StageNumber)
       docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.9-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.9-appservice-stage$(StageNumber)
       docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.10-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.10-appservice-stage$(StageNumber)
       docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.11-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.11-appservice-stage$(StageNumber)
+      docker tag $SOURCE_REGISTRY/python:$(PrivateVersion)-python3.12-appservice $TARGET_REGISTRY/python:$(TargetVersion)-python3.12-appservice-stage$(StageNumber)
```

Comment:
We have been having issues with this lately. Could you please split this file into doing sets of 4 and docker pruning between? https://github.com/Azure/azure-functions-docker/blob/dev/host/4/publish.yml#L653

Reply:
Added docker prune